package.json

```diff
@@ -47,7 +50,9 @@
     "testModules": "lerna run test --ignore=cbioportal-frontend --stream",
     "test:watch": "yarn run test -- --watch",
     "test:debug": "yarn run test -- --debug --browsers=Chrome_with_debugging --single-run=false",
-    "prettierLastCommit": "yarn run prettier --write $(git diff-tree --no-commit-id --name-only -r $(git rev-parse HEAD))",
+    "prettierFixCurrentChanges": "yarn run prettier --write $(git diff HEAD --name-only)",
```

Comment:
nice - it does also seem to ignore things in prettierignore even if you explicitly pass that file to prettier. Sweet